### ISSUE TITLE
wishlist: trimming

### ISSUE THREAD

**Issue by bturkus** [amiaopensource/vrecord#139]
the virtual dub announcement got me thinking that trimming without re-encoding would be a great addition to vrecord, something similar to what @privatezero got going with audiorecorder.

**Comment by marshalleq** [comment]
Yes, this would be fantastic.  I find that the sources I have, with the equipment I have means that I need to add about 5-10 seconds of whatever is on the tape prior to the item I am capturing.  This is because it takes a few seconds for the frame TBC to get in sync, which causes a few frame drops (and unnecessary footage at the beginning) which ultimately could be cut out.  Of course cutting out mistakes at the end would be useful too.

**Comment by libbyhopfauf** [comment]
I couldn't agree more! That would be an amazing feature!
